### ISSUE TITLE
Can't resolve '@choc-ui/chakra-autocomplete'

### ISSUE THREAD

**Issue by Kysluss** [anubra266/choc-autocomplete#186]
After the merge of #179 installing the package gives an error "Can't resolve '@choc-ui/chakra-autocomplete'" even in a barebones install from CRA.

Previously, I had updated the peerDependencies manually for 5.1.5 to see if #179 would fix the framer motion dependency issue and it did.

Steps to reproduce:

Initiate a new app using: npx create-react-app my-app
Install this package: npm i @choc-ui/chakra-autocomplate
Update app.js to contain any code from one of the examples
Execute: npm run start to launch the application

Here is a [CodeSandbox](https://codesandbox.io/s/choc-ui-chakra-autocomplete-module-missing-ftx8lj?file=/package.json) showing the issue

When I tested PR #179,  I had just manually updated the peerDependencies and confirmed it worked.  For some reason, that now seems to have broken the actual installation.  I'll try and work on what's up with this, but hoping it might be easily apparent to you.  First time trying to contribute to a package.

**Comment by Kysluss** [comment]
@anubra266 I'm wondering if this is just an issue with the npm deployment?  I was able to get this working locally, just not with the npm package.

Steps I tried locally:

1. Clone repo locally: git clone https://github.com/anubra266/choc-autocomplete.git
2. Run npm pack - this generate a tgz file

In a brand new project created by npx create-react-app my-app

1. Run npm install path/to/above-created-tgz (if using npm install @choc-ui/chakra-autocomplete, you get initially reported issue)
2. Copy one of the App.js files from examples
3. Run npm run start
